netbeans.apache.org/src/content/download/nb17/index.adoc

```diff
@@ -117,21 +117,11 @@ TIP: The Runtime JDK NetBeans uses does not influence the JDK range projects can
 
 == Building from Source
 
-To build Apache NetBeans {netbeans-version} from source you need:
-
-. A distribution of OpenJDK 11.
-. Apache Ant 1.10 or greater (link:https://ant.apache.org[https://ant.apache.org]).
-
-Once you have everything installed then:
-
 1. Unzip link:{url-download}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip[netbeans-{netbeans-version}-source.zip]
 in a directory of your liking.
 
 [start=2]
-. `cd` to that directory, and then run `ant` to build the Apache NetBeans IDE.
-Once built you can run the IDE by typing `./nbbuild/netbeans/bin/netbeans`
-
-For more details refer to the README.
+. Follow the build instructions in the README
```

Comment:
this page slowly mutated into a README, I think I mentioned that already when we updated the NB 16 dl page that we have too many copies of the same information.

The README should be always up to date since it is part of the release.

I did also update the wiki last week, mentioned the README and removed all version numbers from this page:
https://cwiki.apache.org/confluence/display/NETBEANS/Development+Environment